conf/modules.config

```diff
@@ -443,4 +443,13 @@ if( (params.annotate_tools) && (params.annotate_tools.contains('merge')) ) {
         ]
     }
 
+    withName: 'MULTIQC' {
```

Comment:
there are 2 multiqc scopes now, one also in line 112

Reply:
good catch